src/commands/tick-stop.js

```diff
@@ -0,0 +1,77 @@
+import prompt from 'prompt'
+import moment from 'moment'
+import { parser } from 'tickbin-parser'
+import db from '../db'
+import createEntry from '../create'
+import { writeSaved } from './output'
+
+export default { builder, handler : stop}
+
+function builder(yargs) {
+  return yargs
+  .usage('Usage: tick stop')
+  .example('tick stop', 'stop timer for the current time')
+  .example('tick stop "10am"', 'stop timer for 10am')
+  .example('tick stop "10am creating bugs #dev"', 'stop timer for 10am and provide new commit message')
+}
+
+function stop(argv) {
+  db.get('_local/timers')
+  .then(timersDoc => parseMessage(timersDoc, argv._[1]))
+  .then(commitTimer)
+  .then(writeSaved)
+  .catch(err => console.log(`\nCould not stop your timer\n${err.message}`))
```

Comment:
Not sure if there is a reason for this, but why does `stop` print a newline before the error message but `start` [doesn't?](https://github.com/tickbin/tickbin/pull/270/files#diff-ecd55f42665b0b941f00cf33cc9a3888R29).

Likely noone would notice, so mostly just curious if it was deliberate.